### ISSUE TITLE
Not compatible PlotSquared 4.427

### ISSUE THREAD

**Issue by andris155** [utarwyn/EnderContainers#35]
```
[22:11:00] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to EnderContainers v2.2.0
java.lang.NoSuchFieldError: USE
	at fr.utarwyn.endercontainers.dependency.PlotSquaredDependency.canPlayerUseEnderchest(PlotSquaredDependency.java:71) ~[?:?]
	at fr.utarwyn.endercontainers.dependency.PlotSquaredDependency.onBlockChestOpened(PlotSquaredDependency.java:40) ~[?:?]
	at fr.utarwyn.endercontainers.dependency.DependenciesManager.onBlockChestOpened(DependenciesManager.java:70) ~[?:?]
	at fr.utarwyn.endercontainers.enderchest.EnderChestListener.onPlayerInteract(EnderChestListener.java:77) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor1052.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.14.4.jar:git-Paper-236]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.14.4.jar:git-Paper-236]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.14.4.jar:git-Paper-236]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:545) ~[patched_1.14.4.jar:git-Paper-236]
	at org.bukkit.craftbukkit.v1_14_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:454) ~[patched_1.14.4.jar:git-Paper-236]
	at net.minecraft.server.v1_14_R1.PlayerInteractManager.a(PlayerInteractManager.java:443) ~[patched_1.14.4.jar:git-Paper-236]
	at net.minecraft.server.v1_14_R1.PlayerConnection.a(PlayerConnection.java:1361) ~[patched_1.14.4.jar:git-Paper-236]
	at net.minecraft.server.v1_14_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:27) ~[patched_1.14.4.jar:git-Paper-236]
	at net.minecraft.server.v1_14_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:5) ~[patched_1.14.4.jar:git-Paper-236]
	at net.minecraft.server.v1_14_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:23) ~[patched_1.14.4.jar:git-Paper-236]
	at net.minecraft.server.v1_14_R1.TickTask.run(SourceFile:18) ~[patched_1.14.4.jar:git-Paper-236]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.14.4.jar:git-Paper-236]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.14.4.jar:git-Paper-236]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.14.4.jar:git-Paper-236]
	at net.minecraft.server.v1_14_R1.MinecraftServer.aX(MinecraftServer.java:1029) ~[patched_1.14.4.jar:git-Paper-236]
	at net.minecraft.server.v1_14_R1.MinecraftServer.executeNext(MinecraftServer.java:1022) ~[patched_1.14.4.jar:git-Paper-236]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeAll(IAsyncTaskHandler.java:95) ~[patched_1.14.4.jar:git-Paper-236]
	at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:1158) ~[patched_1.14.4.jar:git-Paper-236]
	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:925) ~[patched_1.14.4.jar:git-Paper-236]
	at java.lang.Thread.run(Unknown Source) [?:?]
```

**Comment by utarwyn** [comment]
I don't understand why authors of PlotSquared are changing the API at each release :cry: I will work on that, no problem haha...

**Comment by utarwyn** [comment]
Oh sorry, I just figured out that you are using a **development build of PlotSquared**, and that causes an error because of breaking changes. Because I would like to be sure that my plugin works for the majority, I don't want to support these breaking builds.

**Maybe we should wait for a stable version to be released?** If you really want to use this version of PlotSquared, you will have to adapt my plugin yourselves, sorry.

**Comment by andris155** [comment]
Please add option to Endercontainers config disable plugins dependency if don't work or don't need.
Example:
Dependency:
  Comment what do this if false.
  PlotSquared: true
  Comment what do this if false.
  Worldguard: false
Etc...

**Comment by utarwyn** [comment]
Yep, nice feature to be implemented in the plugin, I definitly agree with your idea. I will work on that, thank you for all your feedbacks to improve the plugin! 💯

**Comment by andris155** [comment]
PlotSquared new version released.
https://www.spigotmc.org/resources/plotsquared.1177/updates

**Comment by utarwyn** [comment]
This error should be fixed with my commit 4ddfe6e9e80de0eb1486db44c9641f0af1110d59. 😄

**Comment by andris155** [comment]
Thanks
